hub/management/commands/import_mp_job_titles.py

```diff
@@ -0,0 +1,84 @@
+from django.core.management.base import BaseCommand
+
+import pandas as pd
+from tqdm import tqdm
+
+from hub.models import Area, DataSet, DataType, Person, PersonData
+
+CONSTITUENCY_CORRECTIONS_DICT = {
+    "Beverly and Holderness": "Beverley and Holderness",
+    "Brighton Pavilion": "Brighton, Pavilion",
+    "Enfield Southgate": "Enfield, Southgate",
+    "Lewisham Deptford": "Lewisham, Deptford",
+    "Richmond": "Richmond (Yorks)",
+    "Na h-Eileanan an Lar": "Na h-Eileanan an Iar",
+}
+
+
+class Command(BaseCommand):
+    help = "Import MP Job titles"
+
+    def handle(self, quiet=False, *args, **options):
+        self._quiet = quiet
+        self.import_results()
+
+    def get_df(self):
+        df = pd.read_csv("data/mp_job_titles.csv", usecols=["Constituency", "Title"])
+        df = df.query(
+            "not(Title.str.contains('Member of Parliament for') | Title.str.contains('MP for'))"
```

Comment:
might also be worth stripping "Member of $foo Select Committee" here too, given we're getting that data elsewhere. Worth keeping the Chair bit, unless we can update the select committee importer to pull in that data.

Reply:
This could be a challenge, if Select Committees are ever listed _alongside_ other useful positions, in the same string. (Does that ever happen?)

Reply:
> (Does that ever happen?)

Ah, answered my own question – no it doesn’t (currently).

So yeah, could strip anything that matches `r'^Member.*Select Committee$'`. Feels like keeping the handful of "Chair of […] Select Committee" references is useful, because we don’t currently differentiate between chairs and members of select committees elsewhere on the page.